tests/browser.js

```diff
@@ -245,6 +246,17 @@ describe('Routing', function() {
       });
     });
 
+     it('navigates to external Urls', function(done) {
+      assertRendered('mainpage');
+      clickOn(app.refs.externalUrl);
+      delay(function() {
+        // Not sure how to vefify this case and continue.
+        // Browser gets redirected to external.
```

Comment:
Best way to test this is probably to wrap `Link`'s `onClick` handler. Something like:

```javascript
wrappedLinkHandler = function(e) {
  var ret = oldHandler(e);
  assert(ret);
  assert(!e.defaultPrevented)
}